tests/integration/pilot/common/traffic.go

```diff
@@ -210,7 +211,7 @@ func (c TrafficTestCase) Run(t framework.TestContext, namespace string) {
 		if len(c.config) > 0 {
 			cfg := yml.MustApplyNamespace(t, c.config, namespace)
 			// we only apply to config clusters
-			t.ConfigIstio().YAML("", cfg).ApplyOrFail(t)
+			t.ConfigIstio().YAML("", cfg).ApplyOrFail(t, apply.NoCleanup)
```

Comment:
was this for debugging on intended to merge?

Reply:
Should be `CleanupConditionally` - was having issues with the test suite nuking stuff out from under me.